source/views/settings/sections/login-credentials.js

```diff
@@ -50,7 +50,7 @@ export class CredentialsLoginSection extends React.Component {
           label='Username'
           _ref={ref => this._usernameInput = ref}
           disabled={loading}
-          onChangeText={text => text && this.props.onChangeUsername(text)}
+          onChangeText={(text='') => this.props.onChangeUsername(text)}
```

Comment:
To note: we only had to set the default value here to satisfy Flow. As far as we can tell, it would always be a string anyway, but Flow disagreed, so here we are.